From ace42061df8f11c651bf54b66c26a0a8860a7843 Mon Sep 17 00:00:00 2001
From: tstegema <tstegema@90b65887-3827-0410-9a23-83215b262276>
Date: Wed, 25 Jun 2014 11:49:41 +0000
Subject: [PATCH 0476/1000] MANTIS 0032175 extension SteT Mantis 0032175:
 Conveyor: conveyor3dev send image of baggage to application ConveyorDev.idl
 V1.3

added type imageList for IND_PIECE_IMAGE

git-svn-id: svn://localhost/SelfServiceCommon/trunk@690 90b65887-3827-0410-9a23-83215b262276
---
 SelfServiceCommon/Massai/idl/Massai/ConveyorDev.idl | 8 ++++++--
 1 file changed, 6 insertions(+), 2 deletions(-)

diff --git a/SelfServiceCommon/Massai/idl/Massai/ConveyorDev.idl b/SelfServiceCommon/Massai/idl/Massai/ConveyorDev.idl
index e82411d7..1eaea5a8 100644
--- a/SelfServiceCommon/Massai/idl/Massai/ConveyorDev.idl
+++ b/SelfServiceCommon/Massai/idl/Massai/ConveyorDev.idl
@@ -39,14 +39,15 @@
 **/
 module conveyordev
 {
-   const string interface_version = "1.2";
+   const string interface_version = "1.3";
 /*
 using DeviceComponents::indicationListener and DeviceComponents::Generic from mdci.idl
   RC indicationListener::indication(in any ind);
     with ind: DeviceComponents::indication
       with ind.type:   conveyordev::indicationType
       with ind.status: conveyordev::status
-      with ind.data:   conveyordev::bagInfo                 for IND_PIECE_WEIGHT, IND_PIECE_BARCODE, IND_PIECE_IMAGE        
+      with ind.data:   conveyordev::bagInfo                 for IND_PIECE_WEIGHT, IND_PIECE_BARCODE
+      with ind.data:   conveyordev::imageList               for IND_PIECE_IMAGE        
       with ind.data:   conveyordev::characteristics         for IND_CHARACTERISTICS_INFO 
       with ind.data:   conveyordev::extendedCharacteristic  for IND_EXTENDED_CHARACTERISTICS_INFO 
       with ind.data:   DeviceComponents::nullData           for the others
@@ -264,6 +265,9 @@ using DeviceComponents::indicationListener and DeviceComponents::Generic from md
       StringList  baggageBarcodes;
    };
 
+   typedef sequence<octet> image;
+   typedef sequence<image> imageList;
+
    /**
     *
     * Definition of command types passed to this device handler.
-- 
2.41.0.windows.1

